src/main/webapp/script.js

```diff
@@ -652,14 +729,74 @@ function navigateGraph(amount) {
  * Assumes currGraphNum is between 0 and numMutations
  */
 function updateButtons() {
-  if (currGraphNum === 0) {
+  // The use of <= and >= as opposed to === is for safety
+  if (Math.floor(currMutationIndex) <= -1) {
     document.getElementById("prevbutton").disabled = true;
   } else {
     document.getElementById("prevbutton").disabled = false;
   }
-  if (currGraphNum === numMutations) {
+  if (currMutationIndex >= mutationIndexList.length - 1) {
     document.getElementById("nextbutton").disabled = true;
   } else {
     document.getElementById("nextbutton").disabled = false;
   }
+  const numElement = document.getElementById("num-mutation-display");
+  numElement.innerText = `Graph ${currMutationNum + 1}`;
```

Comment:
I think it might still be valuable to display "Graph {currMutation} of {totalMutation}"

Reply:
for the total, should that be the total number of relevant mutations there are? because if so, would it be weird if it said like `Graph 29 out of 10` (this would happen if there were 10 mutations relevant to a node searched, but it was graph 29 overall since we're displaying the mutationNum rather than index).

Reply:
I just added something with the total number of graphs in parenthesis! please follow up if something else would be better!

Reply:
I was thinking of displaying for example `Graph 29 out of 3000` where the total number of mutations is displayed. To do this you'll have to send an additional piece of information over from the server.